services/processor/javascript/test/e2e/deploy_javascript_test.go

```diff
@@ -25,7 +25,9 @@ func TestDeploymentOfJavascriptContract(t *testing.T) {
 		t.Skip("Skipping E2E tests in short mode")
 	}
 
-	t.Skip("not implemented")
+	if !jsEnabled() {
```

Comment:
why not put the whole file under build flag?

Reply:
Because there are two different things here: under `javascript` build flag primarily runs the dummy plugin test. It ensures that the interface is doing ok and does not require an actual JS plugin.

To enable tests against actual plugin, I introduce `JS_ENABLED` variable in my tests.